react/externals.json

```diff
@@ -53,7 +53,11 @@
       },
       {
         "clientOnly": true,
-        "path": "npm:lazysizes@5.1.0/lazysizes.min.js"
+        "path": "npm:lazysizes@5.2.2/plugins/unveilhooks/ls.unveilhooks.min.js"
```

Comment:
Takes up 1.4kb, but allows lazyloading background images, which might make up for it